### PR TITLE
Update address on event card component

### PR DESCRIPTION
**Pull request by joshuanatanielnm**

### CODE REVIEW

File: src/components/events/event-card.tsx [hunk 1]
```diff
@@ -29,21 +31,26 @@ const EventCard: React.FC<EventCardProps> = ({ event }) => {
...
+        <div className="mb-8">
+          <h6 className="mb-1">Venue</h6>
+          {event.venue.name}, {event.venue.address}
           <br />
-          {event.venue.latitude}, {event.venue.longitude}
         </div>
 
+        <A className="block mb-2" href={googleMapsUrl}>
+          Open in Google Maps
+        </A>
```
Comment (grikomsn) [inline src/components/events/event-card.tsx:52]:
```suggestion
        <A className="block mb-2" href={googleMapsUrl}>
          View in Google Maps
        </A>
```

**Comment by grikomsn** [comment]
@febriansyahdotid ini fix ya? Atau masih ada perubahan?

**Comment by grikomsn** [comment]
Masih ongoing atau gimana, @febriansyahdotid?

<details>
<summary><b>Screenshot preview</b></summary><br />

image

</details>

**Comment by febriansyahdotid** [comment]
@grikomsn @joshuanatanielnm @ssembara maybe you can improve

**Comment by grikomsn** [comment]
Closed due to inactivity and upcoming website rework.

**Pull request closed without merging**